dbms/src/Server/Server.cpp

```diff
@@ -1410,22 +1410,19 @@ int Server::main(const std::vector<std::string> & /*args*/)
         global_context->setMinMaxIndexCache(minmax_index_cache_size);
 
     /// Size of max memory usage of DeltaIndex, used by DeltaMerge engine.
-    /// This setting is currently a bit tricky:
     /// - In non-disaggregated mode, its default value is 0, means unlimited, and it
-    //    controls the number of total bytes keep in the memory.
-    /// - In disaggregated mode, its default value is 2000. 0 means cache is disabled, and it
-    ///   controls the **number** of trees keep in the memory. <-- Will be fixed.
+    ///   controls the number of total bytes keep in the memory.
+    /// - In disaggregated mode, its default value is memory_capacity_of_host * 0.02.
```

Comment:
```suggestion
    /// - In disaggregated mode, its default value is memory_capacity_of_host * 0.02.
    ///   0 means cache is disabled.
```
https://github.com/pingcap/tiflash/blob/bccc345b8a82f1c9c96aeaf400557e252073bfd9/dbms/src/Interpreters/SharedContexts/Disagg.cpp#L75-L83